device/ospfv2/router.yaml

```diff
@@ -0,0 +1,177 @@
+components:
+  schemas:
+    Device.Ospfv2:
+      description: >-
+        A container of properties for an OSPFv2 router and its interfaces & Route Ranges.
+      type: object
+      required: [interfaces, name]
+      properties:
+        name:
+          x-include: ../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        choice:
```

Comment:
I think router-id should be object and inside that we should have a choice and sub-fields. Current way looks wrong for choice.